include/unk.h

```diff
@@ -0,0 +1,19 @@
+#ifndef UNK_H
+#define UNK_H
+
+#include "ultra64.h"
+
+#define UNK_TYPE s32
+#define UNK_TYPE1 s8
+#define UNK_TYPE2 s16
+#define UNK_TYPE4 s32
+#define UNK_TYPE8 s64
+#define UNK_PTR void*
+#define UNK_RET UNK_TYPE
+#define UNK_ARGS 
+#define UNK_FUN_ARG void(*)(void)
+//#define UNK_FUN_PTR(name) void(*name)(void)
+typedef UNK_RET (*UNK_FUN_PTR)(UNK_ARGS);
```

Comment:
Funny little story.
I originally took this file from MM for my drmario64 repo, and changed this define.
Then I used it for puzzleleague and af, and now it is here.

I just think it is funny that this file with this useless comment is moving around between repos lol

You can keep the comment if you like lol